pkg/image/importer/importer.go

```diff
@@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
 			importDigest.Err = err
 			continue
 		}
+		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
```

Comment:
I'd rather we calculcate the size only when it's not there. Not all registries we use are hub.docker.com, for that cases (where v1 schema is supported) it's unnecessarily calculates the size. 

This could be done as a followup to this issue, we care to unblock the queue, atm.